src/bhost_fpga.py

```diff
@@ -22,19 +22,10 @@
 
 class FpgaBHost(FpgaHost):
```

Comment:
The way it was implemented the correlator would keep a collection of X-hosts and B-hosts, which would imply two katcp connections to each FPGA. Thought it might be cleaner to simply have a separate class that uses the xhosts (i.e. don't inherit, compose). I would probably also rename them to something else (perhaps BEngineManager?) to avoid the idea that they are host objects.

Another approach would be to have the BHosts subclass xhosts, and then have the correlator object instantiate them instead of xhosts as needed. Personally I perfer to avoid long inheritance chains and mega-objects with millions of methods.